src/Microsoft.Framework.Configuration/Internal/ConfigurationFocus.cs

```diff
@@ -6,77 +6,95 @@
 
 namespace Microsoft.Framework.Configuration.Internal
 {
-    public class ConfigurationFocus : IConfiguration
+    public class ConfigurationFocus : IConfigurationSection
```

Comment:
Will need some refactoring but I think this should be collapsed into ConfigurationSection now.